src/runtime_src/core/edge/user/aie/aie.h

```diff
@@ -20,22 +20,54 @@
 #define xrt_core_edge_user_aie_h
 
 #include <vector>
+#include <queue>
 
+#include "core/edge/common/aie_parser.h"
 extern "C" {
 #include <xaiengine.h>
 }
 
 namespace zynqaie {
 
+#define BD_HIGH_ADDR_MASK               0xFF00000000L
+#define BD_LOW_ADDR_MASK                0xFFFFFFFF
+
+#define GET_BD_HIGH_ADDR(addr)          (((addr) & BD_HIGH_ADDR_MASK) >> 32)
+#define GET_BD_LOW_ADDR(addr)           ((addr) & BD_LOW_ADDR_MASK)
```

Comment:
I am not positive how you imagine these being used.   I am wary of # defines which are controlled by preprocessor and not scoped to namespace.  It is better if these are inline functions, something like:
```
inline uint64_t
get_bd_high_addr(uint64_t addr)
{
    constexpr uint64_t hi_mask =  0xFF00000000L
    return (addr & hi_mask) >> 32;
}
```
Now if these functions are used only in graph.cpp, then maybe just declare and define them there?